frontend/src/views/Inbox.vue

```diff
@@ -220,12 +228,10 @@ export default {
 
       this.get_tickets()
     },
-    toggleView () {
-      this.list = !this.list
-    },
     togglePersonal () {
-      this.showPersonal = !this.showPersonal
-      this.get_tickets()
+      axios.post('/api/inboxes/1/tickets/1/subscribe')
```

Comment:
klopt dit in de togglePersonal?

Reply:
Oeps, nee dat was vanwege een test, goeie catch!